provider/ansible/module.rb

```diff
@@ -48,10 +48,17 @@ def nested_obj_dict(prop, object, properties, spaces)
 
       # Returns an array of all base options for a given property.
       # rubocop:disable Metrics/CyclomaticComplexity
+      # rubocop:disable Metrics/PerceivedComplexity
+      # rubocop:disable Metrics/AbcSize
       def prop_options(prop, _object, spaces)
+        default = if prop.default_value && prop.is_a?(Api::Type::Integer)
```

Comment:
Would it be worth trying to have something similar to https://github.com/GoogleCloudPlatform/magic-modules/blob/master/google/golang_utils.rb?

Reply:
I really like that. 

What do you think about moving that over to provider/core.rb? Something like `quote_value`? I'd like to try and have this functionality shared between Terraform + P/C/A (even though it's super small)